src/ansible_compat/runtime.py

```diff
@@ -165,12 +184,14 @@ def install_collection(
         cmd.append(f"{collection}")
 
         _logger.info("Running %s", " ".join(cmd))
-        run = self.exec(cmd)
+        run = self.exec(
+            cmd,
+            retry=self.retry,
```

Comment:
It feels weird having to pass this flag to both the class initializer and the individual method.